src/index.html

```diff
@@ -7,6 +7,39 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+   
+    <div class="calendar calendar--mounth-length--31 calendar--start-day--mon">
+      <div class="calendar__cell calendar__cell--content-1"></div>
```

Comment:
no need modifiers for calendar__cell
you can make pseudo element for calendar__cell